required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -495,39 +496,59 @@
 %    \item[\socket{tagsupport/fnmark} (1 argument)]
 %
 %      \DescribeSocket[noprint]{tagsupport/fnmark}
-%
+% 
+%    The socket is used in \cs{@footnotemark}/\cs{fnote_footnotemark:} 
+%    and takes \cs{@makefnmark} as argument. It prints the mark in the text
+%    and surrounds it with a tagging structure and a link. As such it is 
+%    not solely for tagging and so should not be used with \cs{UseTaggingSocket}
+%    as this would swallow the argument and loose the link support. 
+%    
```

Comment:
given the documentation the socket should not be "tagsupport/..." should it?

perhaps it should contain sockets that have the tagging stuff inside?

Reply:
Yes, I think one will have to rethink that. But I would suggest to leave that for after the november release and to improve now only the label tagging.

Reply:
ok fine, but then I think we should clearly state at the very top that the socket names as well as the suckets are likely to change because of that

Reply:
I added a comment and a Todo.